.gitignore

```diff
@@ -1,5 +1,5 @@
 $ cat .gitignore
 # exclude
-*.jar
```

Comment:
I needed to allow for the compaction file jars hence the more forgiving filter :)